File: solutions/go/darts/1/darts.go
package darts

func Score(x, y float64) int {
	var a float64 = (x*x) + (y*y)
    if a>100{
        return 0
    }else if a>25{
        return 1
    }else if a>1{
        return 5
    }else{
        return 10
    }
}

File: solutions/c/darts/1/darts.c
#include "darts.h"

uint8_t score(coordinate_t landing_position){
    float d = (landing_position.x*landing_position.x)+(landing_position.y*landing_position.y);
    if (d<=1){
        return 10;
    }else if (d <= 25){
        return 5;
    }else if (d <= 100){
        return 1;
    }
    return 0;
}
